provider/k8s.go

```diff
@@ -1531,8 +1531,12 @@ func (k8s *K8sOfflineStore) GetTrainingSet(id ResourceID) (TrainingSetIterator,
 	return fileStoreGetTrainingSet(id, k8s.store, k8s.logger)
 }
 
-func (k8s *K8sOfflineStore) GetTrainingSetTestSplit(id ResourceID, testSize float32, shuffle bool, randomState int) (TrainingSetIterator, TrainingSetIterator, func() error, error) {
-	return nil, nil, nil, fmt.Errorf("not Implemented")
+func (k8s *K8sOfflineStore) CreateTrainTestSplit(def TrainTestSplitDef) (func() error, error) {
+	return nil, fmt.Errorf("not Implemented")
```

Comment:
same as above. swap to `NewInternalError` and add which method isn't implemented.